src/components/Uploader/index.js

```diff
@@ -99,7 +99,7 @@ class Uploader extends Component {
       instructions
     } = this.props
     const isPoA = !!poaDocumentType
-    const documentTypeGroup = getDocumentTypeGroup(poaDocumentType || documentType)
+    const iconType = getDocumentTypeGroup(poaDocumentType || documentType) || 'identity'
```

Comment:
I don't fully understand this part.... can you give a quick 1 line explanation for a JS sdk newbie? Why was showing the wrong icon, and why does it solve?

Reply:
Before the code was only handling documents, but we also need to show document for face. I will refactor this to add a css class called `face` which behaves the same as `identity` and I think it will make more sense